docs/user-guide/how-to-deploy.rst

```diff
@@ -0,0 +1,51 @@
+:desc: How to deploy your Rasa Assistant with Docker or Kubernetes
+
+.. _deploying-your-rasa-assistant:
+
+Deploying your Rasa Assistant
+===============================
+
+.. edit-link::
+
+This page explains when and how to deploy an assistant built with Rasa. 
+It will allow you to make your assistant available to users and set you up with a production-ready environment.
+If you haven't used Rasa before, we'd recommend that you start with the :ref:`rasa-tutorial`.
```

Comment:
is this sentence necessary? I feel like people navigating to this page will have started with rasa already

Reply:
sure, something similar is on the tutorial pages but people would be going there first so it's not necessary here

Reply:
removed